### PR TITLE
docs(appium): add sponsorship program information

### PR DESCRIPTION
**Pull request by jlipps**
Proposing a sponsorship program and compensation scheme for the project.

Needs approval from @KazuCocoa @mykola-mokhnach @eglitise at least. Happy to work through more details here on this PR.

### CODE REVIEW

File: GOVERNANCE.md
Comment (jlipps) [inline GOVERNANCE.md:18]: in this section just doing some cleanup of former maintainers

File: GOVERNANCE.md
Comment (jlipps) [inline GOVERNANCE.md:30]: proposing a substantive change here to better reflect the reality when we have so few maintainers. it's (IMO) unreasonable to expect that in a group of 4 maintainers, 2 of them could not belong to the same company. But in a group of, say, 9, we certainly wouldn't want 4 of them to be.

File: GOVERNANCE.md
Comment (jlipps) [inline GOVERNANCE.md:55]: taking the opportunity to clean up this list as well. feel free to suggest other modifications that i have missed.

**Review by KazuCocoa — COMMENTED** [review]
lg overall. the list of commiters is also fine.

Let me ping @Dor-bl here to share this with him (to appreciate his dotnet work mostly)

File: GOVERNANCE.md
Comment (mykola-mokhnach) [reply GOVERNANCE.md:55]: do we have a list of honourable mentions that we keep forever? I would prefer people that ever were part of Appium development to still be mentioned, especially these that contributed a lot.

File: GOVERNANCE.md
Comment (mykola-mokhnach) [inline GOVERNANCE.md:109]: Does github have a possibility to create a filter of matching PRs made within a month or a dashboard, so we could automate the estimation process?

Comment (mykola-mokhnach) [reply GOVERNANCE.md:109]: also not sure how open if the estimation process should be.
Perhaps, we could simply add appropriate tags to PRs to make that more explicit

Comment (eglitise) [reply GOVERNANCE.md:109]: I agree, tags seem like a reasonable option here

File: GOVERNANCE.md
Comment (eglitise) [reply GOVERNANCE.md:55]: Yes, I also think it's better to have a section for former contributors/honorable mentions, instead of removing them

**Comment by mykola-mokhnach** [comment]
By the way we also have people that are active and try to help others at the Appium forum, also doing that for quite a while. I would say we may also encourage them to continue doing this:
- https://discuss.appium.io/u/aleksei/summary
- https://discuss.appium.io/u/wreed/summary

File: GOVERNANCE.md
Comment (Dor-bl) [inline GOVERNANCE.md:88]: Maybe it will be nice to group the committers per project (in case they don't work on multiple ones)

File: GOVERNANCE.md
Comment (jlipps) [reply GOVERNANCE.md:55]: yeah, great idea!

File: GOVERNANCE.md
Comment (jlipps) [reply GOVERNANCE.md:73]: the company would need to warrant it to us, basically, and we understand that contributions wax and wane. this is partly why we are looking for more than just contributions as part of this partnership, but rather maintenance responsibilities.

File: GOVERNANCE.md
Comment (jlipps) [reply GOVERNANCE.md:88]: Good idea, maybe I can just allow people to write what they maintain beside their names in brackets. Groups are difficult since some people are in multiple groups.

File: GOVERNANCE.md
Comment (KazuCocoa) [reply GOVERNANCE.md:73]: Sounds reasonable 👍

File: GOVERNANCE.md
Comment (jlipps) [reply GOVERNANCE.md:55]: ok, this is done now.

**Comment by jlipps** [comment]
@mykola-mokhnach 

> By the way we also have people that are active and try to help others at the Appium forum, also doing that for some years. I would say we may also encourage them to continue doing this

Good idea, how do you recommend we do this in this doc?

**Review by KazuCocoa — APPROVED** [review]

**Comment by KazuCocoa** [comment]
Maybe... we could create a new section like appium forum, which is similar to current `### Project Committers` and put them in there...? Just an idea. This can be another PR I think.

**Review by Dor-bl — APPROVED** [review]

File: GOVERNANCE.md
Comment (eglitise) [inline GOVERNANCE.md:62]: mwakizaka, rerorero and tomriddly are listed both here and in the Emeritus section, is that okay?

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by jlipps** [comment]
> Maybe... we could create a new section like appium forum, which is similar to current ### Project Committers and put them in there...? Just an idea. This can be another PR I think.

Added them in the Contributors section.

File: GOVERNANCE.md
Comment (KazuCocoa) [inline GOVERNANCE.md:57]: Let me modify the list later as:

mwakizaka and rerorero are xcuitest driver and uia2/espresso driver (JS code ones) related.
tomriddly is appium-ios-driver related.


And noticed we could add these persons as well. I'll create a new PR later after this merge for them:
https://github.com/appium/java-client/commits?author=valfirst
https://github.com/appium/appium-espresso-driver/commits?author=pr4bh4sh

Comment (jlipps) [reply GOVERNANCE.md:57]: great, thank you!

**Review by eglitise — APPROVED** [review]

**Pull request merged**
